lambda/data/facts_hi_IN.py

```diff
@@ -0,0 +1,13 @@
+data = [
+    "बुध गृह में एक साल में केवल अठासी दिन होते हैं",
+    "सूरज से दूर होने के बावजूद, Venus का तापमान Mercury से ज़्यादा होता हैं",
+    "Earth के तुलना से Mars में सूरज का size तक़रीबन आधा हैं",
+    "सारे ग्रहों में Jupiter का दिन सबसे कम हैं",
+    "सूरज का shape एकदम गेंद आकार में हैं"
+]
+
+SKILL_NAME = "अंतरिक्ष फ़ाक्ट्स"
+GET_FACT_MESSAGE = "ये लीजिए आपका fact: "
+HELP_MESSAGE = "आप मुझे नया fact सुनाओ बोल सकते हैं या फिर exit भी बोल सकते हैं... आप क्या करना चाहेंगे?"
+HELP_REPROMPT = "मैं आपकी किस प्रकार से सहायता कर सकती हूँ?"
+STOP_MESSAGE = "अच्छा bye, फिर मिलते हैं"
```

Comment:
Why do you have this? I see only en in the skill.json

Reply:
I think those are the data for user's references. Check this line https://github.com/RonWang/skill-sample-python-fact/blob/ask-cli-x/lambda/lambda_function.py#L31. Currently it only uses en-XX but those utterances are for  the users who want to try other languages.